src/Microsoft.AspNet.OData.Shared/Query/Expressions/SelectExpandWrapper.cs

```diff
@@ -58,6 +58,18 @@ public IEdmTypeReference GetEdmType()
             return model.GetEdmTypeReference(elementType);
         }
 
+        public IEdmType GetEdmType(IEdmModel model)
```

Comment:
>GetEdmType [](start = 24, length = 10)

Why can't use above "GetEdmType()" method?

Reply:
Changed it to use the above API.